### PR TITLE
Chars limit on textarea

### PR DESCRIPTION
**Pull request by EmmericIsaiah**
50 min, 500 max, required

### CODE REVIEW

File: Question_form.html [hunk 1]
```diff
@@ -1,54 +1,56 @@
...
-            <label for="Voyage">Voyage</label><br>
-        </form>
-        <script src="Assets/JS/Index.js"></script>
+        <textarea placeholder="Ecrivez votre question ici" id="myid" class="block sizebis" name="ref" required minlength="50" maxlength="500" size="50" wrap="hard"></textarea>
+        <input value="Envoyer" class="button gauche" id="Confirm" type="submit">
+        <input type="button" onclick="window.location.href = 'Index.html'" value="Annuler" class="button droite" id="Cancel" type="reset">
+    </form>
+    <div class="scroll">
+        <h2>Choisir mots-clés :</h2>
+        <input type="radio" id="Finance" name="mots-clés" value="Finance">
```
Comment (2binfree) [inline Question_form.html]: name content must be in english : mots-clés => keyword

File: Question_form.html [hunk 2]
```diff
@@ -1,54 +1,56 @@
...
-            <input type="radio" id="Cuisine" name="mots-clés" value="Cuisine">
-            <label for="Cuisine">Cuisine</label><br>
-            <input type="radio" id="Bricolage" name="mots-clés" value="Bricolage">
-            <label for="Bricolage">Bricolage</label><br>
-            <input type="radio" id="Voyage" name="mots-clés" value="Voyage">
-            <label for="Voyage">Voyage</label><br>
-        </form>
-        <script src="Assets/JS/Index.js"></script>
+        <textarea placeholder="Ecrivez votre question ici" id="myid" class="block sizebis" name="ref" required minlength="50" maxlength="500" size="50" wrap="hard"></textarea>
+        <input value="Envoyer" class="button gauche" id="Confirm" type="submit">
```
Comment (gabfour) [inline Question_form.html]: every name content must be in english
do not mix english and french

File: Question_form.html [hunk 3]
```diff
@@ -1,54 +1,56 @@
 <!DOCTYPE html>
-<html lang="en" dir="ltr">
+<html lang="fr" dir="ltr">
```
Comment (heidisbk) [inline Question_form.html]: use english as language : lang="en"

File: Question_form.html (hunk 2 above)
Comment (gabfour) [inline Question_form.html]: every name content must be in english
do not mix english and french

File: Question_form.html (hunk 3 above)
Comment (EmmericIsaiah) [reply Question_form.html]: Why do you want to put it on "en" ? This website is for french people, doesn't it ? The default text is for french in my opinion.

File: Question_form.html (hunk 2 above)
Comment (EmmericIsaiah) [reply Question_form.html]: This website is for french people. The french words are for people who will navigate to the website.

File: Question_form.html (hunk 1 above)
Comment (EmmericIsaiah) [reply Question_form.html]: Done and changed.

File: Question_form.html (hunk 2 above)
Comment (EmmericIsaiah) [reply Question_form.html]: Names that are in french are text the people will see when they will navigate on our website. Others words are in english.

Comment (gabfour) [reply Question_form.html]: "button gauche"?

**Pull request merged**